en/config/actuators.md

```diff
@@ -140,32 +140,60 @@ Note that you will need to also ensure that the ESC associated with bidirectiona
 - `Type`: The type of each control surface: `LeftAileron`, `RightAileron`, `Elevator`, `Rudder`, `Left Elevon`, `Right Elevon`, `Left V-Tail`, `Right V-Tail`, `Left Flap`, `Right Flap`, `Airbrakes`, `Custom`.
-- `Roll scale`: Effectiveness of actuator around roll axis (normalised: -1 to 1).
+- `Roll Torque`: Effectiveness of actuator around roll axis (normalised: -1 to 1).
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
-- `Pitch scale`: Effectiveness of actuator around pitch axis (normalised: -1 to 1).
+- `Pitch Torque`: Effectiveness of actuator around pitch axis (normalised: -1 to 1).
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
-- `Yaw scale`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).
+- `Yaw Torque`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
 - `Trim`: An offset added to the actuator so that it is centered without input.
   This might be determined by trial and error.
-- `Slew Rate`: Minimum time allowed for the motor/servo signal to pass through the full output range, in seconds.
+- (Advanced) `Slew Rate`: Minimum time allowed for the motor/servo signal to pass through the full output range, in seconds.
   - The setting limits the rate of change of an actuator (if not specified then no rate limit is applied).
      It is intended for actuators that may be damaged if they move too fast — such as the tilting actuators on a tiltrotor VTOL vehicle.
   - For example, a setting of 2.0 means that the motor/servo will not be commanded to move from 0 to 1 at a rate that completes the operation in less than 2 seconds (in case of reversible motors, the range is -1 to 1).
+- (Advanced) `Flap Scale`: How much this actuator is deflected at the "full flaps configuration" [0, 1] (see [Flaps and Spoilers Configuration](#flaps-and-spoilers-configuration) below).
+  Can be used to configure aerodynamic surface as flap or to compensate for generated torque through main flaps.
+- (Advanced) `Spoiler Scale`: How much this actuator is deflected at the "full spoiler configuration" [0, 1] (see [Flaps and Spoilers Configuration](#flaps-and-spoilers-configuration) below).
+  Can be used to configure aerodynamic surface as spoiler or to compensate for generated torque through main spoiler.
 - `Lock control surfaces in hover`:
   - `Enabled`: Most vehicles do not use control surfaces in hover. Use this setting to lock them so that they don't affect vehicle dynamics.
   - `Disabled`: Set this for vehicles that use control surfaces in hover, such as the duo tailsitter (which uses elevons for pitch and yaw control). It should also be set for vehicles that use control surfaces to provide additional stabilization in hover mode when moving at speed or in high winds.
 
+#### Flaps and Spoilers Configuration
```

Comment:
This looks good, except that you're configuring aileron, rudder/elevator, flaps.

We also have the option to configure spoiler as its own actuator (at least in my UI). SHould we at least mention this since we're talking about spoilers?

Reply:
>except that you're configuring aileron, rudder/elevator, flaps.

I'm not talking about flaps/spoilers actuators here, but flaps/spoiler control configuration..that's the thing I tried to make clear below.

And yes, mentioning here that dedicated spoiler actuators are also available would be good.

Reply:
As it is now, the heading is opposite to your intent, which is why it is confusing - it makes me thing this is about configuring flaps, and configuring spoilers. What you mean is a title "Flap Scale and Spoiler Scale Configuration". 

What I think you are saying is that all types of controls might have an _effect_ like flaps (which affect lift) and spoilers (which slow the aircraft). What this section about is defining how affect much each type of control has, specifically for each of those scales. Obviously you would expect dedicate flaps to have a very big effect, so a large flap scale value. Ditto for dedicated spoilers and the spoiler scale value.

So something like:


```suggestion
#### Flap Scale and Spoiler Scale Configuration

Control surfaces such as ailerons (which primarily affect roll) and elevators (which primarily affect pitch), may also have an effect on lift or vehicle speed, in the same way as dedicated flaps and spoiler controls.
The flap scale and spoiler scale allow you to specify how much each control should be deflected when the controller is demanding "full flaps" or "full spoiler".
Then control allocator can then use all the available control surfaces it wants for flap and spoiler control (usually flaps, ailerons, elevator).

```